src/components/ScreenContentWrapper.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Platform, View, ViewProps } from 'react-native';
+import ScreenContentWrapperNativeComponent from '../fabric/ScreenContentWrapperNativeComponent';
+
+export const NativeScreenContentWrapper: React.ComponentType<ViewProps> =
+  Platform.OS !== 'web' ? (ScreenContentWrapperNativeComponent as any) : View;
+
+function ScreenContentWrapper(props: ViewProps) {
```

Comment:
I think so. This is just a ReactViewGroup underneath. The sole purpose of this new view is to enable us to access dimensions of our content & be able to call callbacks when react recomputes layout.

Reply:
Maybe I should exclude layout props from there, I need to think about it.

Reply:
Maybe even I could make this component internal-only, w/o the need to export it as part of public interface. 

The goal is to have screen content's children (so no header config) wrapped in this view & ensuring it is laid out by RN, so that it's layout is consistent with the contents. 

The exclusion of `headerConfig` makes this difficult - I need to wrap children in place where I pass `headerConfig` appropriately, thus the place is native stack view render function -> `react-navigation` has to be aware of this view :/ 

Current conclusion: I must include this in public API -> I must refine the props it can take. Ideally I would want to restrict all layout props, because the implementation relies on fact that it wraps the contents tightly.

Reply:
TODO: Exclude layout props